yarn-project/prover-node/src/factory.ts

```diff
@@ -71,15 +77,20 @@ export async function createProverNode(
 
   const epochCache = await EpochCache.create(config.l1Contracts.rollupAddress, config);
 
-  // If config.p2pEnabled is true, createProverCoordination will create a p2p client where txs are requested
-  // If config.proverCoordinationNodeUrls is not empty, createProverCoordination will create set of aztec node clients from which txs are requested
-  const proverCoordination = await createProverCoordination(config, {
-    aztecNodeTxProvider: deps.aztecNodeTxProvider,
-    worldStateSynchronizer,
+  const proofVerifier = config.realProofs ? await BBCircuitVerifier.new(config) : new TestCircuitVerifier();
```

Comment:
We should probably use the `QueuedIVCVerifier` here to prevent overwhelming the host.